### DISCUSSION TITLE
Menu Headings & Menu Seperators

### DISCUSSION THREAD

**Discussion by southpaw3229** [templaza/astroid-framework#222] [Q&A] [answered]
Has anyone had issues with the menu items (menu heading and menu seperators) acting as menu items.  Typically in joomla, these get treated as just text and non links.  I have sub headers for my menus and the menu headings and menu seperators acts as links and lets the visitor hover over them and it looks like a link that even tries to initiate a javascript function.

Here is my staging site - If you go to GOLF < THE COURSE (this is the menu heading) < Golf Rates
you will notice THE COURSE acts like a link but it's a menu heading. 

Thanks

**Comment by southpaw3229** [comment]
https://golf-course-websites.com/DEV/trinidad/

**Comment by sonvnn** [comment, marked as answer]
Hi @southpaw3229 

You can use this custom css to change style of Menu Heading
```
.nav-item-megamenu .nav-submenu .item-link-heading.item-level-2, .nav-item-megamenu .nav-submenu .item-link-separator.item-level-2 {
    cursor: auto;
}
```

Best Regards,
Sonny

**Comment by southpaw3229** [comment]
Sonny,

Ok thanks.  Any chance the css will be updated in your next release?  This styling is something that is standard in Joomla, but your framework is overriding the joomla css.

Thanks,
Bay

**Reply by sonvnn** [reply]
> Sonny, Ok thanks.  Any chance the css will be updated in your next release?  This styling is something that is standard in Joomla, but your framework is overriding the joomla css. Thanks, Bay
> […](#)
> On Oct 12, 2022, 9:27 PM -0400, templaza/astroid-framework ***@***.***>, wrote: .nav-item-megamenu .nav-submenu .item-link-heading.item-level-2, .nav-item-megamenu .nav-submenu .item-link-separator.item-level-2 { cursor: auto; }

Hello @southpaw3229 ,

This is default style from Astroid and it related to other Astroid Joomla Templates. Please use custom css above to change your style. It related to template style not a bug. So I will not change it in the next update.

Thanks & Best Regards,
Sonny
